server/resp/src/main/java/org/infinispan/server/resp/commands/hash/HMGET.java

```diff
@@ -36,13 +39,19 @@ public HMGET() {
    public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx, List<byte[]> arguments) {
       EmbeddedMultimapPairCache<byte[], byte[], byte[]> multimap = handler.getHashMapMultimap();
       CompletionStage<Collection<byte[]>> cs = multimap.get(arguments.get(0), arguments.subList(1, arguments.size()).toArray(Util.EMPTY_BYTE_ARRAY_ARRAY))
+            .thenApply(this::wrapKeys)
```

Comment:
This looks like a bug with something else?

Reply:
Ah, yeah. I can create an issue/PR for it.

Reply:
Same PR is fine, just separate the commit so it can be tracked independently.

Reply:
Added 6f5e971 for it